smarts/env/gymnasium/driving_smarts_competition_env.py

```diff
@@ -137,25 +139,37 @@ def multi_scenario_v0_env(
     env_specs = _get_env_specs(scenario)
     sstudio.build_scenario(scenario=[env_specs["scenario"]])
```

Comment:
`sstudio` appears undefined.

Reply:
Sorry, I had this commented out locally to test faster.